README.md

```diff
@@ -443,7 +443,15 @@ For more details, review the [globals example](https://github.com/apideck-librar
 
 ## Configure automatic upload to Postman App
 
-To enable automatic uploads of the generated Postman collection through Portman, follow these simple steps:
+> REMARK: Portman does **not** require you to have a Postman account.
+
+In case you want to sync the generated Postman collection with the Postman app ( `portman --syncPostman`), you would need a Postman account since Portman leverages the Postman API to sync the collection.
```

Comment:
```suggestion
In case you want to sync the generated Postman collection with the Postman app (`portman --syncPostman`), you would need a Postman account since Portman leverages the Postman API to sync the collection.
```